app/_src/mesh/features/meshopa.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: MeshOPA (beta) - OPA Policy Integration
+title: MeshOPA - OPA Policy Integration
 content_type: reference
 beta: true
```

Comment:
```suggestion
```
You'll want to remove this too, as leaving it will still generate a beta banner.